.github/workflows/rnmobile-android-runner.yml

```diff
@@ -2,6 +2,9 @@ name: React Native E2E Tests (Android)
 
 on:
     pull_request:
+        types: [review_requested, ready_for_review]
+        paths:
```

Comment:
I don't think that we should use paths here and for iOS, because React Native editor shares code with Gutenberg web.